src/modifiers.rs

```diff
@@ -29,3 +29,25 @@ bitflags! {
         const SUPER = 0x2000;
     }
 }
+
+impl Modifiers {
+    /// Return `true` if the modifiers contain `Modifiers::SHIFT`.
```

Comment:
One nitpick, the comment should be less about the implementation details and more about the function. I'd prefer something like "True if a shift key is pressed."

Reply:
Done!